.drone.star

```diff
@@ -135,7 +157,7 @@ def codestyle():
 				'name': name,
 				'workspace' : {
 					'base': '/var/www/owncloud',
-					'path': 'server/%s' % config['app']
+					'path': 'server/apps/%s' % config['app']
```

Comment:
And there was something different that happened here - I don't remember exactly why this cannot go inside the `apps` folder. But I do know that this is not a normal "real" oC10 app.